app/Http/Controllers/DeviceController.php

```diff
@@ -11,8 +11,11 @@
 use Illuminate\Support\Arr;
 use Illuminate\Support\Facades\Auth;
 use LibreNMS\Config;
+use LibreNMS\Util\Debug;
 use LibreNMS\Util\Graph;
 use LibreNMS\Util\Url;
+use const false;
+use const true;
```

Comment:
What does these do?

Reply:
stupid intellij put those in there...  I'll remove them.